core/src/com/unciv/logic/civilization/Civilization.kt

```diff
@@ -185,6 +185,12 @@ class Civilization : IsPartOfGameInfoSerialization {
      */
     val temporaryUniques = ArrayList<TemporaryUnique>()
 
+    /** Easy way to look up a Civilization's unique units and buildings */
+    @Transient
+    val uniqueUnits = hashSetOf<BaseUnit>()
```

Comment:
Please put these in the cache, we're trying to get rid of transients in the main class :)

Reply:
Done.